content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -226,6 +252,35 @@ curl.exe -LO https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools
 .\PrepareNode.ps1 -KubernetesVersion {{< param "fullversion" >}}
 ```
 
+#### Add Windows Flannel and kube-proxy DaemonSets
+
+Now you can add Windows-compatible versions of Flannel and kube-proxy. In order
+to ensure that you get a compatible version of kube-proxy, you'll need to substitute
+the tag of the image. The following example shows usage for Kubernetes {{< param "fullversion" >}},
+but you should adjust the version for your own deployment.
+
+```bash
+curl -L https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/kube-proxy.yml | sed 's/VERSION/{{< param "fullversion" >}}/g' | kubectl apply -f -
+kubectl apply -f https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/flannel-overlay.yml
```

Comment:
Avoid recommending that readers fetch things using `curl` and then apply those manifests to production clusters.
If there's no reasonable alternative, explain that readers should verify the manifest before they apply it.